source/deploy/try-mattermost.rst

```diff
@@ -0,0 +1,80 @@
+Try Mattermost
+===============
+
+This guide provides instructions for quickly trying out Mattermost using either Docker or Azure Marketplace. These options are ideal for testing and evaluation purposes as they allow you to quickly get a Mattermost instance up and running
+for exploration and testing.
+
+Deployment Options
+------------------
+
+.. tab:: Azure Marketplace
+
+    Mattermost is available as a pre-configured virtual machine image in the Azure Marketplace. This option provides a more production-like environment for testing.
+
+    Prerequisites:
```

Comment:
These headings don't seem to be rendering in the preview environment: 

![image](https://github.com/user-attachments/assets/c48b41e7-1e33-478c-8173-b4382d417ab3)

Reply:
Thanks @cwarnermm!